src/esp/metadata/attributes/ObjectAttributes.cpp

```diff
@@ -96,17 +96,17 @@ ObjectAttributes::ObjectAttributes(const std::string& handle)
 }  // ObjectAttributes ctor
 
 std::string ObjectAttributes::getAbstractObjectInfoInternal() const {
-  return cfg.value("mass")
+  return std::to_string(getMass())
       .append(1, ',')
-      .append(cfg.value("COM"))
+      .append(getAsString("COM"))
       .append(1, ',')
-      .append(cfg.value("inertia"))
+      .append(getAsString("inertia"))
       .append(1, ',')
-      .append(cfg.value("angular_damping"))
+      .append(std::to_string(getAngularDamping()))
       .append(1, ',')
-      .append(cfg.value("linear_damping"))
+      .append(std::to_string(getLinearDamping()))
       .append(1, ',')
-      .append(cfg.value("semantic_id"));
+      .append(std::to_string(getSemanticId()));
 }
```

Comment:
Also :)

Reply:
I'm going to strip out all of the offending functionality from this PR and refactor all of these to use the formatString process in another PR.